core/sr-primitives/src/offchain/http.rs

```diff
@@ -488,6 +488,7 @@ impl<'a> HeadersIterator<'a> {
 	/// Move the iterator to the next position.
 	///
 	/// Returns `true` is `current` has been set by this call.
+	#[allow(clippy::should_implement_trait)]
```

Comment:
do we want to start using clippy flag ?

I can agree on that

Reply:
If `clippy -D clippy::all` becomes a part of CI, it'll be necessary because Clippy isn't always right, or sometimes implementing a lint may do more harm than good.

A flag also signals some information.